generate.go

```diff
@@ -646,8 +648,53 @@ func getDefaultTemplate() specs.Spec {
 			},
 		},
 		Hostname: "shell",
-		Mounts:   []specs.Mount{},
+		Mounts: []specs.Mount{
+			{
+				Destination: "/proc",
+				Type:        "proc",
+				Source:      "proc",
+				Options:     nil,
```

Comment:
On Wed, Mar 23, 2016 at 03:26:02PM -0700, Mrunal Patel wrote:
> > @@ -646,8 +648,53 @@ func getDefaultTemplate() specs.Spec {
> >  			},
> >  		},
> >  		Hostname: "shell",
> > -		Mounts:   []specs.Mount{},
> > +		Mounts: []specs.Mount{
> > +			{
> > +				Destination: "/proc",
> > +				Type:        "proc",
> > +				Source:      "proc",
> > +				Options:     nil,
> 
> @wking runc supports default devices but not mounts yet.

Ah, got it.  Maybe add a note to that effect in the commit message?